src/community/roadmap/index.md.njk

```diff
@@ -0,0 +1,46 @@
+---
+title: Roadmap
+description: This roadmap shows what we’re working on and planning to do.
+section: Community
+layout: layout-pane.njk
+order: 9
+---
+
+This roadmap shows what we’re working on and planning to do.
+
+Some things on the roadmap might change — the purpose is to tell you what’s coming up and help service teams prepare and plan their own work.
+
+See our [Github team board](https://github.com/orgs/alphagov/projects/4) for more details on our plans and day-to-day activities.
+
+## Recently shipped
+
+- Hosted Design System Day on 21 July 2021
+- Implemented Google Analytics on the Design System website.
+- Published checkboxes variant, [none of the above](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-)
+- Reviewed accessibility issue with [conditional reveals](https://accessibility.blog.gov.uk/2021/09/21/an-update-on-the-accessibility-of-conditionally-revealed-questions/)
+- Released [GOV.UK Frontend v3.13.1](https://github.com/alphagov/govuk-frontend/releases/tag/v3.13.1)
+- Published [accessibility acceptance criteria for contributions](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md)
+- Completed a mini discovery on how different teams are prototyping across government
+
+## Working on now
+
+- Revise accordion design contributed by GOV.UK to address an accessibility issue
+- Release [GOV.UK Frontend Version 4.0.0](https://github.com/alphagov/govuk-frontend/issues/2274), includes revised accordion, fixes to headers and footers and a number of breaking changes
+- Host community co-design work to [turn task-list into a component](https://design-system.service.gov.uk/patterns/task-list-pages/)
+- Host community co-design work to make maps better
+- Improve how [users can find out what we are working on](https://github.com/alphagov/govuk-design-system/issues/1410)
+
+## Coming up next
+
+- Release hide this page component
+- Investigate potential changes to browser support
+- Investigate improving Javascript configuration
+- Improve how contributions are prioritised
+- Ensure the Design System meets WCAG 2.2
+
+## Future plans
+
+- Investigate Sass module system (mid 2022)
+- Investigate removing compatibility mode (mid 2022)
+
+<p class="govuk-body govuk-!-font-size-16">Last updated 27 September 2021</p>
```

Comment:
I think this could get a little bit lost or ignored down here – IMO it's fairly important information for interpreting the roadmap. 

You could add it as a new line in the section at the top of the page, before the 'Recently shipped' heading. If we did that I'd probably remove the `govuk-!-font-size-16` class and treat it as a normal paragraph.

Obvs feel free to disagree.

Reply:
Thanks I've done it